File: ticketProject5-DistributedC/TicketSystemCC/model/model/Seller.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{

    public class Seller : Entity<long>
    {
        private string firstName { get; set; }
        private string lastName { get; set; }

        private string username { get; set; }
        private string password { get; set; }

        public Seller(string firstName, string lastName, string username, string password)
        {
            this.firstName = firstName;
            this.lastName = lastName;
            this.username = username;
            this.password = password;
        }

        public string FirstName
        {
            get { return firstName; }
            set { firstName = value; }
        }

        public string LastName
        {
            get { return lastName; }
            set { lastName = value; }
        }

        public string Username
        {
            get { return username; }
            set { username = value; }
        }

        public string Password
        {
            get { return password; }
            set { password = value; }
        }

        public override string ToString()
        {
            return Id + "," + firstName + "," + lastName;
        }
        public override bool Equals(object o)
        {
            if (this == o) return true;
            Seller that = (Seller)o;
            return firstName.Equals(that.firstName) &
                lastName.Equals(that.lastName) &
                username.Equals(that.username) &
                password.Equals(that.password);
        }

    }
}

File: ticketProject6-RMI/TicketSystem/TicketModel/src/main/java/ticket/model/Seller.java
package ticket.model;

import java.io.Serializable;
import java.util.Objects;

public class Seller extends Entity<Long>  implements Serializable {
    private String firstName;
    private String lastName;
    private String password;
    private String username;

    public Seller(String firstName, String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }

    public Seller(String username, String password, boolean a) {
        this.username = username;
        this.password = password;
    }

    public Seller(String firstName, String lastName,String password) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.password = password;
    }

    public Seller(String firstName, String lastName, String username, String password) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.username = username;
        this.password = password;
    }


    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    @Override
    public String toString() {
        return this.getFirstName() + " " + this.getLastName();
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (!(o instanceof Seller)) return false;
        Seller that = (Seller) o;
        return getFirstName().equals(that.getFirstName()) &&
                getLastName().equals(that.getLastName()) &&
                getUsername().equals(that.getUsername()) &&
                getPassword().equals(that.getPassword());
    }

    @Override
    public int hashCode() {
        return Objects.hash(getFirstName(), getLastName(), getUsername(), getPassword());
    }

}
